js/bundle_test.go

```diff
@@ -404,6 +410,141 @@ func TestNewBundleFromArchive(t *testing.T) {
 	assert.Equal(t, "hi!", v2.Export())
 }
 
+func TestOpen(t *testing.T) {
+	var testCases = [...]struct {
+		name           string
+		openPath       string
+		pwd            string
+		isError        bool
+		isArchiveError bool
+	}{
+		{
+			name:     "notOpeningUrls",
+			openPath: "github.com",
+			isError:  true,
+			pwd:      "/path/to",
+		},
+		{
+			name:     "simple",
+			openPath: "file.txt",
+			pwd:      "/path/to",
+		},
```

Comment:
Can you add the following relative test cases, since I'm not sure they would be handled correctly?
```
		{
			name:     "relative1",
			openPath: "to/file.txt",
			pwd:      "/path",
		},
		{
			name:     "relative2",
			openPath: "./path/to/file.txt",
			pwd:      "/",
		},
		{
			name:     "relative wonky",
			openPath: "../path/to/file.txt",
			pwd:      "/path",
		},
```